components/rhs_thread/rhs_thread.tsx

```diff
@@ -172,6 +172,19 @@ export default class RhsThread extends React.Component<Props, State> {
         }
     }
 
+    private userIsAtLastPost = () => {
+        const postReplies = (this.rhspostlistRef.current as HTMLElement)?.childNodes;
```

Comment:
Why do we need to cast this to an HTMLElement? Shouldn't an HTMLDivElement already have these fields?

Maybe this is related to how I keep having issues treating HTMLElement as a base type for the other HTML*Element types though

Reply:
Also, I think `children` might be preferable to `childNodes` here